packages/webapp/src/webview/state/reducers.ts

```diff
@@ -231,6 +238,20 @@ function restartAnswerReducer(newState: AppState): AppState {
     return newState;
 }
 
+/**
+ * Go to home page.
+ *
+ * @param newState - already cloned state that is modified and returned
+ * @returns new state with changes
+ */
+function goToHomePageReducer(newState: AppState): AppState {
+    newState.activeScreen = 'HOME';
+    newState.query = '';
+    newState.activeNodeSharing = null;
+    newState.activeGuidedAnswerNode = [];
```

Comment:
Do we need to clear the query when we navigate to home? 
I think this is candidate for UX review.
By the way, this is now active irrespective of the `betaFeatures` toggle, meaning this changes also the general behavior.

Reply:
from what I understand the only thing that needs to be behind `betaFeatures` is the new home page layout. if `betaFeatures` is off, the old home page will be shown